imdb/views.py

```diff
@@ -0,0 +1,56 @@
+import json
+import pdb
+import sys
+import traceback
+
+import requests
+
+import SearchResults
+
+
+from bs4 import BeautifulSoup
+from django.http import HttpResponse
+from django.shortcuts import render
+from django.core.paginator import Paginator, PageNotAnInteger, EmptyPage
+
+
+
+def search_movie(request):
+
+    movie = request.GET.get('q','')
+    error, status, movie_list = SearchResults.get_search_movies(movie)
+
+    if error:
+        data = json.dumps(movie_list)
```

Comment:
it dumps a json serializable(may be python list or dict, over which you can iterate) into json
json.loads changes the json back to the corresponding python data type